components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8833,9 +8832,10 @@ def getFamilies(self):
         :rtype:     Dict
         """
         if not len(self._qf):
-            for f in [BlitzObjectWrapper(self._conn, f)
-                      for f in self._re.getAvailableFamilies()]:
-                self._qf[f.value.lower()] = f
+            query_service = self._conn.getQueryService()
+            for f in self._conn.getEnumerationEntries('Family'):
+                family = query_service.get('Family', f.id)
```

Comment:
`getEnumerationEntries()` already makes a query to retrieve these families, why are we doing it again here and once for every model?